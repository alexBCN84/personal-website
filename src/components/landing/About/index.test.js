import React from 'react';
import { render } from '@testing-library/react';
import { About } from './index';

describe('About component', () => {
    it('renders without crashinf', () => {
        const { container } = render(<About />)
        expect(container.firstChild).toMatchSnapshot();
    });
});