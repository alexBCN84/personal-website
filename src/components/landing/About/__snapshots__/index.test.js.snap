// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders without crashinf 1`] = `
<div
  id="about"
  style="background-color: rgb(37, 50, 55); width: 100vw; height: 100vh;"
>
  <div>
    <h1>
      About me
    </h1>
  </div>
</div>
`;
